src/main/java/decodes/hdb/HdbTimeSeriesDAO.java

```diff
@@ -831,23 +829,24 @@ private void doDelete(CTimeSeries ts)
 	{
 		String idstr = "SDI=" + ts.getSDI() + ", intv=" + ts.getInterval();
 		String q =  "{ call delete_from_hdb(?,?,?,?,?,?)}";
-		oracle.jdbc.OracleCallableStatement cstmt = null;
+
 		GregorianCalendar gc = new GregorianCalendar(
 			TimeZone.getTimeZone(DecodesSettings.instance().sqlTimeZone));
-		try
+		try (Connection c = getConnection();
+			 oracle.jdbc.OracleCallableStatement cstmt =
+			 	(OracleCallableStatement)c.unwrap(oracle.jdbc.OracleConnection.class).prepareCall(q);
```

Comment:
Yeah. @agilmore2 will need to test that. The unwrap might not actually be needed here, but he was getting issues with casts so I thought it would be good to leave one example somewhere.

Reply:
Can test this tonight. I had to make some of these changes locally just to get 7.0.10 working for HDB at all. I'll be interested to see how your changes match up. ;)